site/app/controllers/student/TeamController.php

```diff
@@ -165,14 +165,16 @@ public function sendInvitation($gradeable_id) {
             $this->core->redirect($return_url);
         }
 
-        $invite_id = $_POST['invite_id'];
-        if ($this->core->getQueries()->getUserByID($invite_id) === null) {
+        $invite_id = self::cleanInviteId($_POST['invite_id']);
+        $invited_user = $this->core->getQueries()->getUserById($invite_id);
+
+        if (empty($invited_user)) {
             // If a student with this id does not exist in the course...
             $this->core->addErrorMessage("User {$invite_id} does not exist");
             $this->core->redirect($return_url);
         }
 
-        if ($this->core->getQueries()->getUserByID($invite_id)->getRegistrationSection() === null) {
+        if (empty($invited_user->getRegistrationSection())) {
```

Comment:
This change should be reverted back as a comparison to `null` since a student's registration section could be null if they've dropped the class or haven't been assigned to one yet.